src/main/java/shopservice/CsvWriteServiceImpl.java

```diff
@@ -0,0 +1,16 @@
+package shopservice;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class CsvWriteServiceImpl implements WriteService {
```

Comment:
```suggestion
public class FileWriterServiceImpl implements WriteService {
```
in this impl you don't do anything special for csv file. It can be used for any file (theoretically only report will differ)